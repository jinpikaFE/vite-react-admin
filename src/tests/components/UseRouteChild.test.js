import React, {lazy} from 'react';
import { render, cleanup } from '@testing-library/react';
import UseRouteChild from '../../hooks/UseRouteChild';

afterEach(cleanup);

it('should take a snapshot', () => {

  expect(UseRouteChild({
    path: '/home',
    component: lazy(() => import('@/pages/home')),
    routes: [
      {
        path: '/home/s',
        component: lazy(() => import('@/pages/test')),
      },
      {
        path: '/home/2',
        component: lazy(() => import('@/pages/test')),
      }
    ],
  })).toMatchSnapshot();
});
