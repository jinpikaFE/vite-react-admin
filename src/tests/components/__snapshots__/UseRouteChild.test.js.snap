// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<React.Fragment>
  <React.Fragment>
    <Route
      exact={false}
      path="/home/s"
      render={[Function]}
    />
    <Route
      exact={false}
      path="/home/2"
      render={[Function]}
    />
  </React.Fragment>
</React.Fragment>
`;
